[core]/es_extended/client/modules/actions.lua

```diff
@@ -83,20 +95,20 @@ if Config.EnableDebug then
         print('esx:playerJumping')
     end)
 
-    AddEventHandler('esx:playerEnteringVehicle', function(vehicle, seat, netId)
-        print('esx:playerEnteringVehicle', 'vehicle', vehicle, 'seat', seat, 'netId', netId)
+    AddEventHandler('esx:EnteringVehicle', function(vehicle, plate, seat, netId)

```

Comment:
I think you should lowerCamelCase name convention.
for example: esx:enteringVehicle

Reply:
Sure